README.md

```diff
@@ -19,5 +19,40 @@ npm run seed<br>
 
 This will create 5 photos for each room => 500 entries in DB and S3 bucket => ~ 5-10 minutes execution time<br>
 
+# Available Endpoints
+
+ - Retrieve all photos from room:
+ endpoint: `/rooms/:id/getPhotosByRoomId`
+ Request Type: `GET`
+
+
+ - Create photo:
+ endpoint: `rooms/addRoomPhoto`
+ Request Type: `POST`
+ Required Headers: 'Content-Type' : 'application/json'
+ Required Body:
+ ```{

```

Comment:
What do those 3 back ticks mean again? Is it just readability?

Reply:
Yeah in markdown it makes the surrounded text formatted as a code block